test/line_scanner_spec.coffee

```diff
@@ -1,20 +1,20 @@
-sinon = require 'sinon'
-chai = require 'chai'
-sinonChai = require 'sinon-chai'
+sinon = require('sinon')
+chai = require('chai')
+sinonChai = require('sinon-chai')
 expect = chai.expect
 chai.use(sinonChai)
-LineScanner = require '../tools/line_scanner'
+LineScanner = require('../tools/line_scanner')
 
 
-describe 'LineScanner', ->
+describe('LineScanner', ->
 
-  describe '#scan', ->
-    context 'when the line has valid teascript tokens', ->
-      lineScanner = new LineScanner "f :=  -> 'hello' a.move() bool # comment"
+  describe('#scan', ->
```

Comment:
Python tests don't separate blocks with anything but indentation, do you also dislike that style?